README.md

```diff
@@ -7,7 +7,8 @@
 Describe the differences between a SQL and NoSQL DB, and when you might use each.
 
 ```text
-Your answer...
+SQL is a database that is called as Relational Database.
+NoSQL is a database that is non-relational.
```

Comment:
:+1:

In the future, I recommend you say a little more about the differences.  See our solution branch for suggestions.